addon/components/rdf-input-fields/paragraph.hbs

```diff
@@ -0,0 +1,3 @@
+<AuHeading @skin={{this.titleSkin}}>
```

Comment:
I wonder if we should create a Heading field component instead which would simply be the `AuHeading` component in field form, similar to how it works in the PropertyGroup. Instead of hardcoding the skin here.

That might be more flexible and potentially allows us to drop the title feature of the PropertyGroup in the future as well.

Reply:
I tried to get it like the PropertyGroup component, instead of returning "6", however it did not seem to work.

```  
   get titleSkin() {
    return this.args.group.options.skin || `${this.level + 1}`;
  }
``` 

I added `form:options` with skin like the PropertyGroup, but
`this.args.group` is undefined.

Reply:
Yea, `@group` is only passed into property groups. It's `@field` for regular form fields. So `this.args.field.options.skin` should work I think.